bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/firmware/FirmwareUpdateProgressInfo.java

```diff
@@ -29,6 +30,8 @@
 
     private boolean pending;
 
+    private int progress;
```

Comment:
We should use Integer object so that this value can be null if a FirmwareUpdateHandler cannot provide detailed information about the progress